lib/profiles/geojson.json

```diff
@@ -1,16 +1,16 @@
 {
     "$schema": "http://json-schema.org/draft-04/schema#",
-    "id": "https://raw.githubusercontent.com/fge/sample-json-schemas/master/geojson/geojson.json#",
+    "id": "http://json-schema.org/geojson/geojson.json#",
```

Comment:
@roll The GeoJSON schema [here](https://raw.githubusercontent.com/fge/sample-json-schemas/master/geojson/geojson.json) has a lot of broken links like http://json-schema.org/geojson/geojson.json.

Any idea if there is another schema for validating GeoJSON?